index.html

```diff
@@ -175,62 +179,66 @@ <h2 class="text-choose text-choose--bussness">
+            <p class="card__desc card__desc--wole">
               Slate helps you see how many more days you need to work to reach
               your financial goal
             </p>
-            <div class="card__img-container flex gap-2 py-2">
+            <div class="card__img-container flex">
               <img src="./asserts/Ellipse 2.png" alt="Elipse" />
               <div>
                 <p class="card-name">Regina Miles</p>
-                <p class="card-name--profession">Designer</p>
+                <p class="card-desc">Designer</p>
               </div>
             </div>
           </div>
           <div class="card card--white">
             <img src="./asserts/star.PNG" alt="" />
-            <p class="card__paragraph card__paragraph--wole">
+            <p class="card__desc card__desc--wole">
               Slate helps you see how many more days you need to work to reach
               your financial goal
             </p>
-            <div class="card__img-container flex gap-2 py-2">
+            <div class="card__img-container">
               <img src="./asserts/Ellipse 2 (1).png" alt="Elipse" />
               <div>
                 <p class="card-name">Regina Miles</p>
-                <p class="card-name--profession">Designer</p>
+                <p class="card-desc">Designer</p>
               </div>
             </div>
           </div>
           <div class="card card--white">
             <img src="./asserts/star.PNG" alt="" />
-            <p class="card__paragraph card__paragraph--wole">
+            <p class="card__desc card__desc--wole">
               Slate helps you see how many more days you need to work to reach
               your financial goal
             </p>
             <div class="card__img-container">
               <img src="./asserts/Ellipse 2 (2).png" alt="Elipse" />
               <div>
                 <p class="card-name">Regina Miles</p>
-                <p class="card-name--profession">Designer</p>
+                <p class="card-desc">Designer</p>
               </div>
             </div>
           </div>
         </div>
       </div>
       <!-- end second card -->
     </div>
-    <div class="">
+    <div class="wrapper">
       <div class="choose-container">
-        <div class="choose-content">
-          <h2 class="text-choose text-choose--meet">Meet Our Team.</h2>
-          <p class="text-paragraph-choose">
+        <div class="choose-container__content">
+          <h2
+            class="choose-container__text-choose choose-container__text-choose--meet"
+          >
+            Meet Our Team.
+          </h2>
+          <p class="choose-container__desc">
             Problems trying to resolve the conflict between the two major realms
             of Classical physics: Newtonian mechanics
           </p>
         </div>
       </div>
       <div class="card-container">
-        <div class="card card--profile------">
+        <div class="card">
```

Comment:
is it card global ? it must be improved , avoid global class as much as possible